project/plugins.sbt

```diff
@@ -20,3 +20,5 @@ addSbtPlugin("com.codecommit" % "sbt-github-packages" % "0.5.2")
 addSbtPlugin("com.github.sbt" % "sbt-ci-release"      % "1.5.10")
 
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.0")
+
+addDependencyTreePlugin
```

Comment:
Added to test where slf4j-api 2.0-alpha was coming from with:
`whatDependsOn org.slf4j slf4j-api 2.0.0-alpha1`
No chance though, it looks kind of a bug because of the weird module dependencies for the moment.